src/app/onion/learning-object-builder/learning-object-builder.component.ts

```diff
@@ -100,10 +100,14 @@ export class LearningObjectBuilderComponent implements OnInit, OnDestroy {
 
   adminMode: boolean;
 
+  showServiceFailureModal = false;
+  adminDashboardURL = 'https://admin.clark.center';
```

Comment:
Why are we hardcoding a URL here?

Reply:
Meant to go back and update. Nice catch!